src/tactics.ml

```diff
@@ -113,3 +113,41 @@ let handle_tactic : sig_state -> Proof.t -> p_tactic -> Proof.t =
       handle_refine (Rewrite.reflexivity tac.pos ps)
   | P_tac_sym           ->
       handle_refine (Rewrite.symmetry tac.pos ps)
+  | P_tac_why3(s)    ->
+      (* get the goal to prove *)
+      let (hypothesis, trm) = Proof.Goal.get_type g in
+      let trm = unfold trm in
```

Comment:
Why do you unfold here? This should be done in translate.

Reply:
Deleted in 9d60b154731ebf77c5a6788b9ac65c938ef04521.